### ISSUE TITLE
Known issues with podcasts

### ISSUE THREAD

**Issue by madeofpendletonwool** [madeofpendletonwool/PinePods#333]
There are currently known issues with these podcasts. Something with their feeds. They won't add:

- [ ] Tested from NPR
- [ ] Hacking Postgres

**Comment by madeofpendletonwool** [comment]
Feel free to comment on this issue with any additional podcasts with issues. I will add them in to dig into

**Comment by wessel01** [comment]
https://www.thedutchhistorian.nl/geschiedenis/podcast adding this one causes the connection to MySQL to be lost and makes pinepods crash.

**Comment by madeofpendletonwool** [comment]
@wessel01 Hi there! Could you send me a link directly to the rss feed for that podcast rather than to their webpage? I can't find where an rss feed directly is in order to try adding.

**Comment by wessel01** [comment]
I found this one: https://feeds.buzzsprout.com/2192816.rss and the one on spotify; https://open.spotify.com/show/7EchOjkrIQUIugJr8QNn2L  But I added it via the search button in Pinepods.

**Comment by wessel01** [comment]
Got another one. Search gave me a result and I selected that one. Searching for it gave me the url below.

https://www.omnycontent.com/d/playlist/33dbd2dc-d464-471d-9feb-abae00330078/02c417da-7875-4b3f-bc76-adfe00ac45fe/8d93de0b-dc66-46df-9e32-adfe00acbe42/podcast.rss

**Comment by madeofpendletonwool** [comment]
Hi @wessel01, what is the workflow you're expecting to get these to add? Adding those rss feeds you just provided seems to work? I'm able to play the audio and they show up okay in the feed. Even on the newest  stable version. Are you expecting them to work in a different way? Am I missing something?

![Image](https://github.com/user-attachments/assets/e79a75ed-a391-421f-a488-77e11c09ff66)

**Comment by wessel01** [comment]
Thanks for the quick response. I think I made a mistake in this one. It was another one causing the error. I removed both and currently it's working. I do get another error causing the container to stop. But I'll put that in a different issue unless it's already known. I'll try to fix that first before I continue spamming you with podcast that appear to be 'faulty'. Sorry for the inconvenience.

**Comment by EldritchGarden** [comment]
I have an issue with one podcast that is only showing 64 episodes

<img width="1187" alt="Image" src="https://github.com/user-attachments/assets/360fff8f-85a6-4235-87f2-79cd424faba3" />
<img width="1224" alt="Image" src="https://github.com/user-attachments/assets/ae24ce37-fc32-4fce-b50f-b93c261680a5" />

The search results show correctly, but after adding only 64 episodes are present. The relevant RSS feed is https://feeds.megaphone.fm/NSR9016694791

I noticed that when I add this podcast, through the the RSS feed link or from search, a 500 server error is logged and it says failed but the podcast is added and only shows 64 episodes.

From my digging it looks like maybe the description for one of the episodes might contain some invalid values or something:

![Image](https://github.com/user-attachments/assets/0c8b1934-01bf-4710-b491-53ae22f974c4)

**Comment by madeofpendletonwool** [comment]
Hey @EldritchGarden, thanks for bringing this up and good find for the fix. Are you using Mariadb/Mysql I assume? I do remember running into a similar issue in the past with Maria, which is why we add on those variables in the compose

``
      MYSQL_COLLATION_SERVER: utf8mb4_unicode_ci
      MYSQL_CHARACTER_SET_SERVER: utf8mb4

**Comment by EldritchGarden** [comment]
Yes I'm using a dedicated MariaDB server, which doesn't use utf8mb4 by default

**Comment by ail1020** [comment]
Pinepods fails to add "Firewalls don't stop dragons" podcast either. Podcast url: https://podcast.firewallsdontstopdragons.com/feed/podcast/

**Comment by bbirney** [comment]
Pinepods also has trouble with "Organized Money" (likely not the collation issue mentioned above as I am running postgres). Url: https://feeds.buzzsprout.com/2412334.rss

**Comment by madeofpendletonwool** [comment]
Hi @bbirney, this issue will be released in an update coming this week. That was due to some header information that needed to be sent to the server that particular podcast resides on.

**Comment by madeofpendletonwool** [comment]
Hi @ail1020, I was also able to fix the firewalls don't stop dragons podcast. That was due to some fairly strict header checks on the server that podcast is on. That should work going forward. An update will be releasing this weekend that's quite large. It will include this fix.

**Comment by BigFabe** [comment]
Hi, I am having troubles with Lage der Nation. Feed: https://feeds.lagedernation.org/feeds/ldn-mp3.xml

After the great app revamp I made the switch from audiobookshelf, where I had trouble with the podcast too. I dont know if it is related but here is the fix, which made it work for me on abs: https://github.com/advplyr/audiobookshelf/issues/3346#issuecomment-2335177670

Apart from that, thanks for your amazing work @madeofpendletonwool. Such a great project and i am really looking forward for whats coming next!

**Comment by HerrFrutti** [comment]
Not sure if this is the right issue, but I'm not finding this podcast via the podcast index [https://podcastindex.org/podcast/6785894](https://podcastindex.org/podcast/6785894)

**Comment by madeofpendletonwool** [comment]
@BigFabe - Wow, that is one weird feed to have those kinds of issues. I'll dig in and get that one sorted. Thanks for the kind words!

@HerrFrutti Interestingly it's because of the &. If you search just for Projekt Versicherung then it works. This is a bug related to: https://github.com/madeofpendletonwool/PinePods/issues/629

I'll be digging in and fixing that

<img width="561" height="792" alt="Image" src="https://github.com/user-attachments/assets/c24fa8ad-f23e-4186-94ec-49a9da59112f" />

**Comment by ail1020** [comment]
@madeofpendletonwool It appears https://podcast.firewallsdontstopdragons.com/feed/podcast/ is broken again.

**Comment by saddle-gaudier-06** [comment]
I'm having an issue with "The Common Man Progrum" podcast (link below). I can search for it, find it, and subscribe to it. I also see all the episodes on the podcast homepage in Pinepods. However, after subscribing to the podcast, not all episodes are appearing.

https://www.omnycontent.com/d/playlist/98a2599f-cd9b-4f2a-9be4-b22c00e585f2/50aa602c-e337-4139-a66c-b28901186be7/144e9f92-8dff-407d-8f01-b28901186c1a/podcast.rss

**Comment by brandonb927** [comment]
The Darknet Diaries+ premium URL throws `403` errors when attempting to download anything, this has been happening for months without realizing it 😅 It does not use username/password for auth. What's strange is the latest episode (at the time of writing) works, however any historical download is met with a `403` error.

<img width="1488" height="970" alt="Image" src="https://github.com/user-attachments/assets/fb5a17d7-8f64-4faa-a574-9a1b24c0aab0" />

**Comment by rudism** [comment]
I also had the error downloading Darknet Diaries+. After some testing, I think it's because that podcast is hosted behind Amazon Cloudfront which seems to be deflecting the default `reqwest` library requests. Modifying the download request with `Accept` and `User-Agent` headers fixed it for me. I made a PR with the change [here](https://github.com/madeofpendletonwool/PinePods/pull/802).

**Comment by WiredFire** [comment]
538 Dance Department has an issue with “missing” episodes. 

When searching for the podcast by name within Pinepods the show appears and initially shows 100 episodes including from this week. However after following the podcast only 7 episodes show, most recent being from December 2025

No filtering is in place in the podcast view once added. 

<img width="1170" height="2370" alt="Image" src="https://github.com/user-attachments/assets/8be4d07b-6051-48d4-9770-a7df246926c7" />
<img width="1170" height="2172" alt="Image" src="https://github.com/user-attachments/assets/93e603fc-99fb-47ee-9630-30cadbcf18bd" />
